Marlin/src/feature/spindle_laser.h

```diff
@@ -77,6 +91,33 @@ class SpindleLaser {
   static inline void enable_forward() { set_direction(false); set_enabled(true); }
   static inline void enable_reverse() { set_direction(true); set_enabled(true); }
 
+  #if ENABLED(LASER_POWER_INLINE)
+    // Force disengage planner power control
+    static inline void inline_disable() { planner.settings.laser_status = planner.settings.laser_status & ~1; }
+
+    // Inline modes of all other functions; all enable planner inline power control
+    static inline void inline_enabled(const bool enable) { inline_power(enable ? SPEED_POWER_STARTUP : 0); }
+
+    static void inline_power(const uint8_t pwr) { 
+      #if ENABLED(SPINDLE_LASER_PWM)
+        inline_ocr_power(translate_power(pwr)); 
+      #else
+        planner.settings.laser_status = planner.settings.laser_status | 1; // Enable planner power control
+        planner.settings.laser_status = planner.settings.laser_status | (!!power & 1) << 1; // Setup planer laser state
+        planner.settings.laser_power = pwr;
+      #endif
+    }
+    
+    static inline void inline_direction(const bool reverse) { UNUSED(reverse); } //TODO is this ever going to be needed
+
+    #if ENABLED(SPINDLE_LASER_PWM)
+    static inline void inline_ocr_power(const uint8_t pwr) { 
+      planner.settings.laser_status = planner.settings.laser_status | 1; // Enable planner power control
+      planner.settings.laser_status = planner.settings.laser_status | (!!power & 1) << 1; // Setup planer laser state
```

Comment:
This is a good place to use the `|=` operator. We prefer `x op= y` over `x = x op y` whenever possible.

Reply:
Fixed by jediminer543@0a00a1cdc6d69d62ffc29f3a77b3ecbeb822eaf6